test/e2e/specs/editor/editor__post-advanced-flow.ts

```diff
@@ -63,27 +63,23 @@ describe( DataHelper.createSuiteTitle( `Editor: Advanced Post Flow` ), function
 
 		it( 'Publish post', async function () {
 			postURL = await editorPage.publish();
-			expect( postURL.href ).toBeDefined();
 		} );
 
 		it( 'Validate post', async function () {
-			const testPage = await browser.newPage();
-			await testPage.goto( postURL.href );
+			await page.goto( postURL.href );
```

Comment:
**comment:non-blocking** - To make sure I understand correctly here - we're using the same browser context instead of a new page within a new context, and this results in the cache being invalidated?

As an alternative idea, could [this StackOverflow](https://stackoverflow.com/questions/68522170/playwright-disable-caching-of-webpage-so-i-can-fetch-new-elements-after-scrollin) post help us disable caching instead?

Reply:
> To make sure I understand correctly here - we're using the same browser context instead of a new page within a new context, and this results in the cache being invalidated?

It's not that the cache is being invalidated – IIUC the page is never cached if viewed by the author (in the same, logged-in session).

Reply:
> As an alternative idea, could [this StackOverflow](https://stackoverflow.com/questions/68522170/playwright-disable-caching-of-webpage-so-i-can-fetch-new-elements-after-scrollin) post help us disable caching instead?

I think it's about the local cache, which is not the issue here. I tested manually with the cache disabled and it didn't work, unfortunately.